app/views/layouts/application.html.haml

```diff
@@ -27,32 +27,34 @@
 
     = stylesheet_link_tag 'application.css', media: :screen, 'data-turbolinks-track' => true
     = stylesheet_link_tag 'print.css', media: :print
-
+    = stylesheet_link_tag 'https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css'
```

Comment:
Das sollten wir vielleicht in unser Projekt einbinden, im Falle dass der Link von ihnen ändert.